### PR TITLE
Update DB unit tests to hit test DB

### LINKED ISSUE #237: Create in-memory test DB for unit tests
Currently the browser backend unit tests mock out all SQL functionality. @mweiden [suggested using an in-memory DB](https://github.com/chanzuckerberg/dcp-prototype/pull/235#discussion_r402701183) to test SQL while preserving sub-second unit test times.

### PR DESCRIPTION
**Pull request by calvinnhieu**
- Update `db_utils.py` and `test_db_utils.py` to run against the in-memory DB

Closes #237

### CODE REVIEW

File: dcp_prototype/backend/browser/code/common/db_utils.py [hunk 1]
```diff
@@ -93,7 +139,21 @@ def query_file(self, file_id):
         :param file_id: File ID
         :return: File query result
         """
-        return self._get(File, file_id)
+        file = self._get(File, file_id)
```
Comment (Bento007) [inline dcp_prototype/backend/browser/code/common/db_utils.py:142]: Why do you need to pass in File when you are importing it at the top?

Comment (calvinnhieu) [reply dcp_prototype/backend/browser/code/common/db_utils.py:142]: The first parameter of `_get` specifies which table to query

File: dcp_prototype/backend/browser/code/common/db_utils.py [hunk 2]
```diff
@@ -22,17 +23,62 @@
 class DbUtils:
     def __init__(self):
         self.session = DBSessionMaker().session()
+        self.engine = self.session.get_bind()
 
-    def _get(self, table, entity_id):  # noqa
+    def _get(self, table, entity_id):
```
[the remark this answers is not in the document]
Comment (calvinnhieu) [reply dcp_prototype/backend/browser/code/common/db_utils.py:28]: Sounds good, will do!

**Comment by mweiden** [comment]
LGTM! Given `DbUtil. _parse_multivalue`, I take it you found some gotchas with the new test?

**Pull request merged**